ishell.go

```diff
@@ -495,31 +496,44 @@ func (s *Shell) multiChoice(options []string, text string, init []int, multiResu
 				selected = toggle(selected, cur)
 			}
 		}
-		update()
+		select {
```

Comment:
Also, you don't have to worry about it being blocked as the goroutine on line 519 is always gonna receive.

Wrapping it on a `go func(){ ... }()` is the true way of making it asynchronous. But I do not see the need for that.